mlrun/artifacts/plots.py

```diff
@@ -173,3 +173,48 @@ def get_body(self):
         from bokeh.resources import CDN
 
         return file_html(self._figure, CDN, self.key)
+
+
+class PlotlyArtifact(Artifact):
+    """
+    Plotly artifact is an artifact for saving Plotly generated figures. They will be stored in a html format.
+    """
+
+    kind = "plotly"
+
+    def __init__(
+        self, figure, key: str = None, target_path: str = None,
+    ):
+        """
+        Initialize a Plotly artifact with the given figure.
+
+        :param figure:      Plotly figure ('plotly.graph_objs.Figure' object) to save as an artifact.
+        :param key:         Key for the artifact to be stored in the database.
+        :param target_path: Path to save the artifact.
+        """
+        super().__init__(key=key, target_path=target_path, viewer="plotly")
+
+        # Validate input:
+        try:
+            from plotly.graph_objs import Figure
+        except (ModuleNotFoundError, ImportError) as Error:
+            raise Error(
+                "Using 'PlotlyArtifact' requires plotly package. Use pip install mlrun[plotly] to install it."
```

Comment:
You're mentioning `mlrun[plotly]` but I didn't see you added anything in `setup.py`

Reply:
Nvm I see it's already there